packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 42.4.0 - February 22, 2018
+### Added
+#### salesforcedx-vscode-core
+
+* Include source links to apex classes on apex test failures ([PR #308](https://github.com/forcedotcom/salesforcedx-vscode/pull/308))
```

Comment:
I'll also add another bullet point here:

* "SFDX: Re-Run Last Invoked Apex Test Class" and "SFDX: Re-Run Last Invoked Apex Test Method" show up in the command palette after you have run a test to make it easier to re-run your test. ([PR #308](https://github.com/forcedotcom/salesforcedx-vscode/pull/308))